### PR TITLE
Added Fullscreen mode

### PR DESCRIPTION
**Pull request by Xphalnos**
I added the possibility of putting shadPS4 in Fullscreen. it works but I would need help to make the link between `SDL_SetWindowFullscreen(window, SDL_TRUE);` and the config.toml file. I removed `SDL_SetWindowResizable(window, SDL_FALSE);` because it is initially set to FALSE.

### CODE REVIEW

**Comment by raphaelthegreat** [comment]
Oh nice to have this here. You are quite close, simply replace SDL_TRUE with Config::isFullscreenMode() and you should be set

**Comment by Xphalnos** [comment]
It works !

**Comment by Xphalnos** [comment]
Ready for review !

File: src/common/config.cpp [hunk 1]
```diff
@@ -102,6 +108,7 @@ void load(const std::filesystem::path& path) {
             auto general = generalResult.unwrap();
 
             isNeo = toml::find_or<toml::boolean>(general, "isPS4Pro", false);
+            isFullscreen = toml::find_or<toml::boolean>(general, "Fullscreen", true);
```
Comment (wheremyfoodat) [inline src/common/config.cpp:111]: This setting should ideally have the same capitalization as other settings (first letter non-capital)

File: src/sdl_window.cpp [hunk 2]
```diff
@@ -17,7 +18,7 @@ WindowSDL::WindowSDL(s32 width_, s32 height_, Input::GameController* controller_
         UNREACHABLE_MSG("Failed to initialize SDL video subsystem: {}", SDL_GetError());
     }
 
-    const std::string title = "shadps4 v" + std::string(Common::VERSION);
+    const std::string title = "shadPS4 V." + std::string(Common::VERSION);
```
Comment (wheremyfoodat) [inline src/sdl_window.cpp]: "v0.0.4" probably looks a lot more natural than "V.0.0.4"

File: src/common/config.cpp (hunk 1 above)
Comment (Xphalnos) [reply src/common/config.cpp:111]: Yes but this is made up of just one word

**Pull request merged**